postalicious/app.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express')
+const app = express()
+
+app.use(express.static(__dirname))
```

Comment:
I recommend you change this to point to a directory names `public` so that you are not also serving `/app.js` from your web server.